common/build.gradle

```diff
@@ -93,6 +93,7 @@ dependencies {
     testImplementation "org.powermock:powermock-classloading-xstream:$rootProject.ext.powerMockVersion"
 
     // Instrumented dependencies
+    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
```

Comment:
Can we add this to the version.gradle file? (Most likely we'll need the same thing for ADAL)

Reply:
Yep - once this is fully working I'll tune it up for cleanliness

Reply:
@rpdome I am going to merge this as-is for now and will take you suggestion as a follow-up PR into `android-complete`